site/components/layout/HomeFooter.js

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link'
+import { useTranslations } from 'next-intl'
+
+import SvgContainer from '../svg/SvgContainer'
+
+const HomeFooter = function () {
+  const t = useTranslations()
+  return (
+    <div className="border-slate-300/55 flex items-center justify-between mt-8 p-8 text-gray-500 border-t">
+      <div>
+        <p>{t('copyright')}</p>
+      </div>
```

Comment:
I think we can remove this div
```suggestion
      <p>{t('copyright')}</p>
```

Reply:
Fixed